TCAT/Views/RouteDiagram.swift

```diff
@@ -405,10 +438,10 @@ class RouteDiagram: UIView {
     }
 
     private func resizeHeight() {
-        let firstStopDot = routeDiagramElements[0].stopDot
-        let lastStopDot = routeDiagramElements[routeDiagramElements.count - 1].stopDot
+        let firstStopLabel = routeDiagramElements[0].stopLabel
```

Comment:
I had a crash here because 0 didn't exist on my build. It's better to do `if let variable.first?.stopDot`. Which I did temporary on mine. But if you can maybe investigate why that would be happening, sometimes some merged bus icons would look funny. Something to add to the list post-Sunday beta

Reply:
yeah just tell me what you did to make it crash. it is strange that it crashes b/c a route's directions should have at least 2 directions